src/types/index.ts

```
@@ -32,22 +32,22 @@ export type Country = {
   ccn3?: string;
   cca3: string;
   cioc?: string;
-  independent?: boolean;
+  independent: boolean;
   status: string;
   unMember: boolean;
   currencies?: { [key: string]: { name: string; symbol: string } };
-  idd: { root?: string; suffixes?: string[] };
+  idd: { root: string; suffixes: string[] } | {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)